templates/workspace_services/guacamole/guacamole-server/docker/version.txt

```diff
@@ -0,0 +1 @@
+__version__ = "0.3.6"
```

Comment:
shouldn't it also be 0.4.0?

Reply:
No, only core components follow the main repo tags. Other templates should have versioning done as it makes sense and I thought this is a small change so just a minor version bump.